### PR TITLE
show download Limits on page; Fixes #661

### LINKED ISSUE #661: [Enhancement] Show number of remaining downloads
[Affected versions]:
- Firefox 57.0 and up

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Steps to reproduce]:
1. Start the Firefox browser with and navigate to "https://send.stage.mozaws.net/" page.
2. Upload a file and set it to expire after 20 downloads.
3. Navigate to the homepage and observe the newly uploaded file in the file info table.

[Expected result]:
- The number of remaining downloads is displayed in the table.

[Actual results]:
- No such information is displayed.

[Notes]:
- This information could also be displayed on the file's download page.

### PR DESCRIPTION
**Pull request by shikhar-scs**
The original PR was filed at #681.
However due to some poor rebasing and squashing habits, the final set of commits messed up a bit. 
All the fine details and conversations can be followed at #681. This PR can be referred for what was done in the files.

And I am a 100% sure that this only contains the intended changes.

And thus, this would be the final PR.

@dannycoates please review this .
If correct, please close #681 
:smile:

### CODE REVIEW

**Comment by dannycoates** [comment]
Thank you @shikhar-scs, nice job! 😄

**Comment by shikhar-scs** [comment]
Welcome @dannycoates. Thanks to you for not giving up on me and guiding me so far :smile:

**Comment by himanish-star** [comment]
Congrats, @shikhar-scs. Nice

File: public/locales/en-US/send.ftl [hunk 1]
```diff
@@ -41,6 +41,8 @@ deleteFileButton = Delete file
 sendAnotherFileLink = Send another file
 // Alternative text used on the download link/button (indicates an action).
 downloadAltText = Download
+downloadsFileList = Downloads
+timeFileList = Time
```
Comment (flodolo) [inline public/locales/en-US/send.ftl]: Sorry, I lost track of this PR. This should have a comment, "time" it's a very broad word in English.

Comment (shikhar-scs) [reply public/locales/en-US/send.ftl]: This `Time` refers to this header here 
screenshot from 2018-01-23 12-23-16


### This initially was set as `Expires In` should I change it back.

Comment (flodolo) [reply public/locales/en-US/send.ftl]: That's really not my call ;-)

In the meantime I opened a PR to add a localization comment.

**Pull request merged**